src/examples/trade_assets.js

```diff
@@ -9,7 +9,7 @@ async function tradeAssets() {
   // Fund the wallet's default address with ETH from an external source.
   // Trade 0.00001 ETH to USDC
   const trade = await wallet.createTrade(0.00001, Coinbase.assets.Eth, Coinbase.assets.Usdc);
-  console.log(`Trade successfully completed: `, trade);
+  console.log(`Trade successfully completed: `, trade.toString());
```

Comment:
is there a reason we are not using template literals on these? 

ref - https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html

Reply:
Yeah, I think string casting might be causing the error within the template literal, according to Vignesh's feedback. Passing it as an argument could be a safer approach 👍